esti/gc_test.go

```diff
@@ -61,45 +170,32 @@ func TestCommittedGC(t *testing.T) {
 	if blockstoreType != block.BlockstoreTypeS3 {
 		t.Skip("Running on S3 only")
 	}
-
-	step := viper.GetViper().GetString("gc_step")
-	switch step {
-	case "pre":
-		t.Run("pre", testPreCommittedGC)
-	case "post":
-		t.Run("post", testPostCommittedGC)
-	default:
-		t.Skip("No known value (pre/post) for gc_step")
-	}
-}
-
-func testPreCommittedGC(t *testing.T) {
 	ctx := context.Background()
-	blockstoreType := viper.GetViper().GetString("blockstore_type")
-	fileRefPerTestCase := make(map[string]string)
-
-	for _, testCase := range testCases {
-		t.Run(fmt.Sprintf("Test case %s", testCase.id), func(t *testing.T) {
-			fileExistingRef := prepareForGC(t, ctx, testCase, blockstoreType)
+	for _, tst := range testCases {
+		tst := tst
```

Comment:
why like this?

Reply:
https://gist.github.com/posener/92a55c4cd441fc5e5e85f27bca008721
I will add a comment

Reply:
funny stuff...
a disguised bug... we would normally just pass the variable like in:
```go
for _, v := range arr {
    go func(val <ver's type>) {
        <do something with val>
    }(v)
}
```
thanks